src/devtools/store.js

```diff
@@ -482,37 +482,39 @@ export default class Store extends EventEmitter {
 
           i = i + 3;
 
+          if (__DEBUG__) {
+            if (this._idToElement.has(id)) {
+              console.error(
+                'Store already contains fiber ' + id + '. This is a bug.'
+              );
```

Comment:
We should probably just throw here, since this is no longer expected to ever actually happen.